modules/router/templates/default-vhost.conf.erb

```diff
@@ -5,7 +5,12 @@ server {
     # This is the varnish upstream defined in the www.gov.uk vhost
     proxy_pass http://varnish;
   }
-
```

Comment:
This line will be removed, and all Nginx processes reloaded

Reply:
Updated. Good point about everything reloading.